test/verify/machineslib.py

```diff
@@ -1576,6 +1583,9 @@ def assertScriptFinished(self):
 
             return self
 
+        def assertDomainDefined(self):
+            self.browser.wait(lambda: self.machine.execute("virsh list --persistent --all | wc -l") == '4\n')
```

Comment:
"4" is a bit magical here. How about using `--name`, which skips the table headers, separator lines, etc? Unfortunately it still prints an empty trailing line, so I think `virsh list --persistent --all --name | wc -w` is best (counting words). Then this should be exactly 1, right?

Reply:
I will check for the Vm name in the virsh ouput. I think it's even better.